### PR TITLE
Switch to permanently expand players pane

### PR DESCRIPTION
**Pull request by bradytrappett**

### CODE REVIEW

File: client/src/views/Game.vue [hunk 1]
```diff
@@ -58,6 +58,9 @@
           <!-- Game stats -->
           <v-row>
             <v-card class="ma-3 pa-6" outlined tile>
+              <p>
+              <v-switch v-model="switch1" :label="`Minimize Players`"></v-switch>
```
Comment (logan-hendricks) [inline client/src/views/Game.vue:62]: hmmm.. on my thoughts. I think the label probably would be good to emphasize you are locking the pane.. Like `Lock Players Menu`. But that's just me.

File: client/src/views/Game.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
     >
       <!-- Game Players Drawer -->
       <v-navigation-drawer
-          :expand-on-hover="true"
+          :expand-on-hover="switch1"
```
Comment (logan-hendricks) [inline client/src/views/Game.vue:8]: If possible, I would name this variable something closer to the label on the toggle?  for example, I mentioned the label to be Lock Players Menu. You could call it `lockPlayersMenu` accordingly.

**Comment by jak103** [comment]
Was there are reason this wasn't merged in?

**Comment by bradytrappett** [comment]
I did make two pull requests accidentally, I thought one of them was merged
in.

**Comment by logan-hendricks** [comment]
@jak103 @bradytrappett accidently created two Pull requests. and the second was merged in. This branch and PR should probably be deleted.

**Comment by jak103** [comment]
Verified it is there.

**Pull request closed without merging**